### PR TITLE
chore: add e2e test for crane as registry

### PR DESCRIPTION
**Pull request by thesayyn**
closes #101

### CODE REVIEW

File: .bcr/presubmit.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 bcr_test_module:
-  module_path: "e2e/smoke"
+  module_path: "e2e/crane_as_registry"
```
Comment (alexeagle) [inline .bcr/presubmit.yml:2]: I don't think we should change this, we use "smoke" in all our rulesets as the e2e that BCR presubmit runs. Can we just green up that one on windows?

Comment (thesayyn) [reply .bcr/presubmit.yml:2]: I could switch places. make smoke test use crane and the new test cover zot

Comment (thesayyn) [reply .bcr/presubmit.yml:2]: on second thought, that would make us recommend crane as the default registry, not sure if that's what we want. i don't want users memory to be filled with big blobs

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -23,7 +23,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        folder: [".", "e2e/custom_registry", "e2e/wasm", "e2e/smoke"]
+        folder:
+          [
```
Comment (alexeagle) [inline .github/workflows/ci.yaml:27]: nit: I think a yaml multi-line list should just use the hyphen syntax like
```
folder:
  - .
  - e2e/...
  - ...
```

**Review by alexeagle — APPROVED** [review]

File: .bcr/presubmit.yml (hunk 1 above)
Comment (thesayyn) [reply .bcr/presubmit.yml:2]: done

**Pull request merged**